src/libraries/System.Threading.RateLimiting/src/System/Threading/RateLimiting/NoopLimiter.cs

```diff
@@ -9,14 +9,24 @@ namespace System.Threading.RateLimiting
     internal sealed class NoopLimiter : RateLimiter
     {
         private static readonly RateLimitLease _lease = new NoopLease();
+        private static readonly RateLimiterStatistics _stats = new RateLimiterStatistics()
+        {
+            CurrentAvailablePermits = long.MaxValue,
+            CurrentQueuedCount = 0,
+            TotalFailedLeases = 0,
+            TotalSuccessfulLeases = 0,
```

Comment:
I realize that the no-op limiter might see the biggest perf degradation by accounting for it, but it seems wrong not to implement `TotalSuccessfulLeases`. It could be part of a larger `PartitionedRateLimiter` and having 0 `TotalSuccessfulLeases` for a resource that was accessed a lot would be weird.